parm/archive/gfswave.yaml.j2

```diff
@@ -0,0 +1,18 @@
+    gfswave:
+        {% set head = "gfswave.t" + cycle_HH + "z." %}
+        name: "GFSWAVE"
+        target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gfswave.tar"
+        required:
+            {% for fh in range(0, FHMAX_HF_WAV + FHOUT_HF_WAV, FHOUT_HF_WAV) %}
+            # NOTE This is as explicit as possible without major logic to parse wavepostGRD.
+            #      Matches files of the form "gfswave.tCCz.<region>.<res>.fHHH.grib2".
+            - "{{ COM_WAVE_GRID | relpath(ROTDIR) }}/{{ head }}*.*.f{{ '%03d' % fh }}.grib2"
+            - "{{ COM_WAVE_GRID | relpath(ROTDIR) }}/{{ head }}*.*.f{{ '%03d' % fh }}.grib2.idx"
+            {% endfor %}
+            {% for fh in range(FHMAX_HF_WAV + FHOUT_WAV, FHMAX_WAV_GFS + FHOUT_WAV, FHOUT_WAV) %}
+            - "{{ COM_WAVE_GRID | relpath(ROTDIR) }}/{{ head }}*.*.f{{ '%03d' % fh }}.grib2"
+            - "{{ COM_WAVE_GRID | relpath(ROTDIR) }}/{{ head }}*.*.f{{ '%03d' % fh }}.grib2.idx"
+            {% endfor %}
+            - "{{ COM_WAVE_STATION | relpath(ROTDIR) }}/{{ head }}bull_tar"
```

Comment:
If    DOIBP_WAV='YES' in https://github.com/NOAA-EMC/global-workflow/blob/develop/parm/config/gfs/config.wavepostsbs#L16 

Then we'll also need to archive: 
gfswave.t12z.ibpbull_tar  gfswave.t12z.ibpcbull_tar  gfswave.t12z.ibp_tar  

This isn't turned on in tests but we'll turn it back on eventually.  This is only something that's done for GFS not GEFS or GDAS etc.

Reply:
Alright, I added those product to `gfswave.yaml.j2` wrapped within a `{% if DOIBP_WAV %}`.  Rerunning the `gfsarch` job ran successfully without attempting to archive these files.

Reply:
Thanks @DavidHuber-NOAA ! This is nice and it's great to see an example of how to add a configuration If yes/no to one of these files as well.